themes/dco/templates/individual--foaf-person.ftl

```diff
@@ -49,6 +49,9 @@
                 
         <!-- Websites -->
         <#include "individual-webpage.ftl">
+        
+        <!-- Nationality -->
+        <!-- Nationality: ${homeCountry!}  -->
```

Comment:
Same comment as above. Just in case engagement comes back and wants us to add the home country to the top of the profile, leave it commented out for now.